clouddrift/haversine.py

```diff
@@ -109,3 +110,53 @@ def bearing(
     )
 
     return theta
+
+
+def position_from_distance_and_bearing(
+    lat: float, lon: float, distance: float, bearing: float
+) -> Tuple[float, float]:
+    """Return elementwise new position in degrees from arrays of latitude and
+    longitude in degrees, distance in meters, and bearing in radians, based on
+    the spherical law of cosines.
+
+    The formula is:
+
+    φ2 = asin( sin φ1 ⋅ cos δ + cos φ1 ⋅ sin δ ⋅ cos θ )
+    λ2 = λ1 + atan2( sin θ ⋅ sin δ ⋅ cos φ1, cos δ − sin φ1 ⋅ sin φ2 )
+
+    where (φ, λ) is (lat, lon) and θ is bearing, all in radians.
+    Bearing is defined as zero toward East and increasing counter-clockwise.
```

Comment:
I truly mean increasing here :)

Reply:
increasing counterclockwise is correct but a bearing is an angle not a rotation rate. As such, it is either positive or negative with respect to East. I think _positive counterclockwise_ is therefore the correct wording.

Reply:
I don't follow this. A change of an angle is a rotation, and clockwise is a rotation. An angle can't be clockwise (whether positive or negative) because it's not a rotation, but it can increase clockwise, because an increase of an angle is a rotation. That said, I'll implement your wording of choice.

Reply:
Ok we're debating semantics, which is fun. But yes, my preference is _positive counterclockwise_.

Reply:
Let's keep these suggestions unresolved until they're implemented, otherwise, they get hidden.

Reply:
Sorry, can't seem to do anything right ;)